src/MentionsTextInput.js

```diff
@@ -526,12 +569,17 @@ export default class MentionsTextInput extends Component {
     return !text1 && text2.length > 1 || text1.length < text2.length - 1;
   }
 
+  hasNewLineChar(text) {
+    return text && text.length > 0 && text[text.length - 1] == '\n'
+              || text.indexOf('\n') !== -1;
```

Comment:
These two checks at the end seem redundant, it looks like you could get rid of the `text[text.length - 1] == '\n'`?